src/lua/utils.h

```diff
@@ -81,6 +81,19 @@ extern uint32_t CTID_DATETIME;
 /** Type ID of struct interval. */
 extern uint32_t CTID_INTERVAL;
 
+/**
+ * Creates a new Lua state with a custom allocator function.
+ * Guarantees the state is not NULL. It panics if lua_State can't be allocated.
```

Comment:
Nit: Comment line length is more than 66 symbols.

Reply:
Oh, it's related even to the docs. Fixed.